lib/steps.js

```diff
@@ -56,21 +56,29 @@ Before(async (scenario) => {
   scope.scenario_name = scenario.pickle.name.replace(/[^A-Za-z0-9]/gi, '');
   scope.downloadComplete = false;
   scope.toDownload = '';
+  scope.device = 'pc';
+  scope.activate = click_with[ scope.device ];
+  scope.filename_suffix = '';
 });
 
+// Add a check for an error page before each step? After each step?
+
 // I start the interview on mobile
 // I start the interview on pc
```

Comment:
Should remove these examples now that device is its own step.

Reply:
Good eye!